tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/LocaleCompletionCounter.java

```diff
@@ -28,9 +30,25 @@ public LocaleCompletionCounter(CLDRLocale cldrLocale, STFactory stFactory) {
         Dashboard.setFiles(args, cldrLocale, stFactory);
     }
 
+    /**
+     * Get a baseline count
+     */
+    public LocaleCompletionCounter(CLDRLocale cldrLocale, Factory factory, boolean isBaseline) {
+        this.isBaseline = true;
+        localeId = cldrLocale.toString();
+        level = StandardCodes.make().getTargetCoverageLevel(localeId);
+        lcr = new LocaleCompletion.LocaleCompletionResponse(level);
+        final SurveyMain sm = CookieSession.sm;
+        vv = new VettingViewer<>(sm.getSupplementalDataInfo(), factory, new STUsersChoice(sm));
+        final EnumSet<VettingViewer.Choice> set = VettingViewer.getLocaleCompletionCategories();
+        args = new VettingViewer.DashboardArgs(set, cldrLocale, level);
+        args.setUserAndOrganization(0, VettingViewer.getNeutralOrgForSummary());
+        Dashboard.setFilesForBaseline(args, cldrLocale, factory);
+    }
```

Comment:
a future improvement would be to share more code between the two constructors  -- it's only the first and last lines that differ, right? seems like the first constructor should just call the second one with isBaseline false

Reply:
that's true. 

There's a lot of code churn here now (in fact, merge conflicts during development) so I went for the copy-paste simpler option first